collect_app/src/main/res/raw/settings_schema.json

```diff
@@ -0,0 +1,339 @@
+          "type": "string"
+        },
+        "font_size": {
+          "type": "string",
+          "enum": [
+            "13",
+            "17",
+            "21",
+            "25",
+            "29"
+          ]
+        },
+        "navigation": {
+          "type": "string",
+          "enum": [
+            "swipe",
+            "buttons",
+            "swipe_buttons"
+          ]
+        },
+        "showSplash": {
+          "type": "boolean"
+        },
+        "splashPath": {
+          "type": "string"
+        },
+        "basemap_source": {
+          "type": "string",
+          "enum": [
+            "google",
+            "mapbox",
+            "osm",
+            "usgs",
+            "stamen",
+            "carto"
+          ]
+        },
+        "google_map_style": {
+          "type": "string",
+          "enum": [
+            "1",
+            "2",
+            "3",
+            "4"
+          ]
+        },
+        "mapbox_map_style": {
+          "type": "string",
+          "enum": [
+            "mapbox://styles/mapbox/light-v10",
+            "mapbox://styles/mapbox/dark-v10",
+            "mapbox://styles/mapbox/satellite-v9",
+            "mapbox://styles/mapbox/satellite-streets-v11",
+            "mapbox://styles/mapbox/outdoors-v11"
+          ]
+        },
+        "usgs_map_style": {
+          "type": "string",
+          "enum": [
+            "topographic",
+            "hybrid",
+            "satellite"
+          ]
+        },
+        "carto_map_style": {
+          "type": "string",
+          "enum": [
+            "positron",
+            "dark_matter"
+          ]
+        },
+        "reference_layer": {
+          "type": "string"
+        },
+        "form_update_mode": {
+          "type": "string",
+          "enum": [
+            "manual",
+            "previously_downloaded",
+            "match_exactly"
+          ]
+        },
+        "periodic_form_updates_check": {
+          "type": "string",
+          "enum": [
+            "every_fifteen_minutes",
+            "every_one_hour",
+            "every_six_hours",
+            "every_24_hours"
+          ]
+        },
+        "automatic_update": {
+          "type": "boolean"
+        },
+        "hide_old_form_versions": {
+          "type": "boolean"
+        },
+        "autosend": {
+          "type": "string",
+          "enum": [
+            "off",
+            "wifi_only",
+            "cellular_only",
+            "wifi_and_cellular"
+          ]
+        },
+        "delete_send": {
+          "type": "boolean"
+        },
+        "default_complete": {
+          "type": "boolean"
+        },
+        "constraint_behaviour": {
+          "type": "string",
+          "enum": [
+            "on_swipe",
+            "on_finalize"
+          ]
+        },
+        "high_resolution": {
+          "type": "boolean"
+        },
+        "image_size": {
+          "type": "string",
+          "enum": [
+            "original",
```

Comment:
This should be `original_image_size` I guess.

Reply:
Hmm it looks like this is wrong in our docs ([here](https://docs.getodk.org/collect-import-export/)) and the admin and general keys don't match in our code. Will have a quick look into that.